### ISSUE TITLE
File Not Found Error

### ISSUE THREAD

**Issue by korikuzma** [ga4gh/vrs-python#52]
I have installed ga4gh.vrs[extras] 0.6.3 release. 

When I run `from ga4gh.vrs import models` I get this error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/ga4gh/vrs/__init__.py", line 28, in <module>
    from ._internal.enderef import vrs_deref, vrs_enref
  File "/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/ga4gh/vrs/_internal/enderef.py", line 3, in <module>
    from .models import class_refatt_map
  File "/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/ga4gh/vrs/_internal/models.py", line 61, in <module>
    _load_vrs_models()
  File "/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/ga4gh/vrs/_internal/models.py", line 56, in _load_vrs_models
    models = build_models(schema_path, standardize_names=False)
  File "/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/ga4gh/core/_internal/jsonschema.py", line 27, in build_models
    builder = pjs.ObjectBuilder(path)
  File "/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/python_jsonschema_objects/__init__.py", line 33, in __init__
    with codecs.open(uri, "r", "utf-8") as fin:
  File "/usr/local/Cellar/python@3.9/3.9.1_8/Frameworks/Python.framework/Versions/3.9/lib/python3.9/codecs.py", line 905, in open
    file = builtins.open(filename, mode, buffering)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/kxk102/.local/share/virtualenvs/variant-normalization-Jh4I8RMk/lib/python3.9/site-packages/ga4gh/vrs/_internal/data/schema/vr.json'

```

I have looked in the package directory and the `ga4gh/vrs/_internal/data` directory is not being created. 

@reece , any thoughts on how to fix this? 

Thanks in advance!

**Comment by larrybabb** [comment]
@theferrit32 @ahwagner any insights on this?

**Comment by korikuzma** [comment]
For now, I'm manually creating these directories and the `vr.json` file. I'm mainly concerned on why this is not being created when I install via pip.

**Comment by reece** [comment]
This is a recurrence of #48, but I'm unsure why it's occurring. Investigating...

**Comment by reece** [comment]
Confirmed. I see the problem but am not sure how to fix it yet.

Two renames happened recently: 1) vr.{yaml,json} → vrs.{yaml,json; 2) vrs-python/vr-spec → vrs-python/submodules/vrs
1 was for naming consistency. 2 was to make it clear that vrs is a submodule. 

It appears that updating subrepo locations is not being handled by git the way other renames would be. Therefore, in 0.6, the subrepo just doesn't exist.

I'll continue to investigate.

**Comment by reece** [comment]
Update: I'm still working on this.

**Comment by reece** [comment]
This is now fixed in 0.6.4.
